siad/gateway_test.go

```diff
@@ -43,7 +43,7 @@ func TestPeering(t *testing.T) {
 	// Create to peers and add the first to the second.
 	peer1 := newDaemonTester(t)
 	peer2 := newDaemonTester(t)
-	peer1.callAPI("/gateway/peer/add?addr=" + string(peer2.netAddress()))
+	peer1.callAPI("/gateway/peer/add?Address=" + string(peer2.netAddress()))
```

Comment:
not a fan of capitalization. It seems uncommon in other APIs, and we don't want to be surprising.

One thing we could do is normalize the query strings to lower case before processing them. I'm not totally sure how that would work though.